index.js

```diff
@@ -1,18 +1,269 @@
+            <div class="card-content">
+              <h2 class="card-title">${projects[i].title}</h2>
+              <p class="text-content">${projects[i].content}
+              </p>
+              <ul class="tags">
+                ${projectTechs.join('')}
+              </ul>
+            </div>
+            <button type="button" class="btn mobile see-project">
+              See Project
+            </button>
+          </div>`;
+};
+
 document.addEventListener('DOMContentLoaded', () => {
-  const menu = document.getElementById('menu-');
-  const button = document.getElementById('mob-barss');
-  const closebtn = document.getElementById('closed');
-  const menuitems = document.getElementById('mlist');
-
-  button.onclick = function showMenu() {
-    menu.style.display = 'block';
-  };
-
-  closebtn.onclick = function hideMenu() {
-    menu.style.display = 'none';
-  };
-
-  menuitems.onclick = function hideMenu() {
-    menu.style.display = 'none';
-  };
+  projects.forEach((ele, i) => {
+    if (i > 0) showCards(i);
+  });
+
+  const seeProject = document.querySelectorAll('.see-project');
+
+  seeProject.forEach((project, i) => {
+    seeProject[i].addEventListener('click', () => {
+      openModal(i);
+      const cancel = document.querySelector('.cancel');
+      cancel.addEventListener('click', () => {
+        modal.classList.toggle('hidden');
+        wrapper.classList.toggle('blur');
+        overFlow.classList.toggle('scroll-off');
+      });
+    });
+  });
+
+  const card = document.querySelectorAll('.card');
+  document.querySelector('.avatar').src = projects[0].imageURL.at(0);
+  card.forEach((element, i) => {
+    element.style.backgroundImage = `url(${projects[i + 1].imageURL[0]})`;
+  });
+});
+
+// Get references to the form fields and reset button
+const fullNameField = document.getElementById('fullName');
+const emailField = document.getElementById('email');
+const messageField = document.getElementById('message');
+const resetButton = document.getElementById('reset');
+
+// Retrieve the saved form data from local storage on page load
+window.addEventListener('load', () => {
+  fullNameField.value = localStorage.getItem('fullName') || '';
+  emailField.value = localStorage.getItem('email') || '';
+  messageField.value = localStorage.getItem('message') || '';
+});
+
+// Reset the form and clear local storage when the reset button is clicked
+resetButton.addEventListener('click', () => {
+  localStorage.clear();
+  fullNameField.value = '';
+  emailField.value = '';
+  messageField.value = '';
+});
+
+submit.addEventListener('click', (event) => {
+  const email = document.getElementById('email').value;
+  const error = document.querySelector('.error');
+  const isLowerCase = (str) => str === str.toLowerCase();
+  if (!isLowerCase(email)) {
+    event.preventDefault();
+    error.innerHTML = 'Email has to be in lowercase!';
+    setTimeout(() => {
+      error.innerHTML = '';
+    }, 10000);
```

Comment:
- [ ] When I enter any details in the form, I get the below error and data is not being saved and loaded from the local storage. Please make sure to fix this issue and when users enter any details in the form it should save on the local storage and on page load users should be able to see the data.

![image](https://user-images.githubusercontent.com/83803414/233143381-12e6279e-7edd-4b07-a4f0-5f694d521243.png)